tests/integration/release/conftest.py

```diff
@@ -154,7 +148,10 @@ def noised_data(
     }
     if dataset_func != generate_social_security:
         kwargs["state"] = state
-    noised_data = profile_data_generation(release_output_dir)(dataset_func)(**kwargs)
+    timestamped_dir = request.config.getoption("--output-dir")
```

Comment:
Don't you need to add the timestamp at this point? So far the output-dir is likely to be simply the release test folder based on the default

Reply:
Oh, I remember. We said that it will be somehow up to each job to create its own timestamped directory or soemthing like that, right? I wonder if we do need one higher-level timestamp here (one extra level of depth)...